cmd/explaintest/r/explain_complex.result

```diff
@@ -105,14 +105,13 @@ PRIMARY KEY (`aid`,`dic`)
-      └─HashAgg_11(Probe)	53.00	cop[tikv]		group by:test.dt.ds, test.dt.p1, test.dt.p2, test.dt.p3, test.dt.p4, test.dt.p5, test.dt.p6_md5, test.dt.p7_md5, funcs:count(test.dt.dic)->Column#32, funcs:firstrow(test.dt.ds2)->Column#34
-        └─Selection_15	66.25	cop[tikv]		ge(test.dt.ds, 2016-09-01 00:00:00.000000), le(test.dt.ds, 2016-11-03 00:00:00.000000)
-          └─TableRowIDScan_14	2650.00	cop[tikv]	table:dt	keep order:false, stats:pseudo
+Sort_8	53.00	root		test.dt.ds2:desc
```

Comment:
Schema should be printed here. Test cases of `explaintest` have not updated yet ?

Reply:
which test could re-record it?

Reply:
@pingyu I did run `cmd/explaintest/run-tests.sh -r all`.It is seem Sort_8's schema is not set from child?